addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -96,9 +96,19 @@
                                     "widget": "contact",
                                     "fields": ["phone", "mobile", "email"]
                                 }'/>
-                                <a t-att-href="event._google_map_link()" target="_blank" class="btn btn-secondary text-bg-secondary w-100">
+                                <!-- To be removed by next saas: temporary hide -->
+                                <a t-att-href="event._google_map_link()" target="_blank" class="d-none btn btn-secondary text-bg-secondary w-100">
                                     <p class="mb-0"><i class="fa fa-map-marker fa-fw" role="img"/>Get the direction</p>
                                 </a>
+                                <!-- ==== -->
+
+                                <a t-att-href="event._google_map_link()" target="_blank" class="btn btn-light" title="Get directions">
+                                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="align-text-top" role="img" style="height: 1em; width: 1em">
+                                        <rect width="24" height="24" opacity="0"/>
+                                        <path role="presentation" fill="currentColor" d="M13.67 22h-.06a1 1 0 0 1-.92-.8l-1.54-7.57a1 1 0 0 0-.78-.78L2.8 11.31a1 1 0 0 1-.12-1.93l16-5.33A1 1 0 0 1 20 5.32l-5.33 16a1 1 0 0 1-1 .68z"/>
+                                    </svg>
+                                    Get directions
+                                </a>
```

Comment:
> So why not modify the existing one?

It's a quite prominent element, I think that's one of the first things you want to xpath to customize the design.
Since the element doesn't have an ID, we have to assume that people are using its classes to target it.

Creating a new element and hide the prev. one sounds safer,  but I can just modify the current one, as you prefer 👍 

> Also would say what we're changing about the design. And might need a .pot
👍

Reply:
To me it's natural to override from the href for this but indeed, can't be sure people didn't use the classes